### ISSUE TITLE
tox-to-nox: no module named jinja2

### ISSUE THREAD

**Issue by jaraco** [wntrblm/nox#760]
### Current Behavior

After pipx install nox, I noticed there's a tox-to-nox command. I thought I'd run it to see what I could learn about the difference between tox's `testenv.setenv` and nox's `env`. So I ran `tox-to-nox` but got an error:

```
 @ tox-to-nox
Traceback (most recent call last):
  File "/Users/jaraco/.local/bin/tox-to-nox", line 7, in <module>
    from nox.tox_to_nox import main
  File "/Users/jaraco/Library/Application Support/pipx/venvs/nox/lib/python3.12/site-packages/nox/tox_to_nox.py", line 23, in <module>
    import jinja2
ModuleNotFoundError: No module named 'jinja2'
```

### Expected Behavior

Probably the installed CLI command should work out of the box.

### Environment

```Markdown
- OS: macOS 14.3.1
- Python: 3.12.1
- Nox: 2023.4.22
```

**Comment by jaraco** [comment]
I see now [tox-to-nox docs](https://nox.thea.codes/en/stable/usage.html#converting-from-tox) direct the user to install the prerequisite extra. So maybe this issue should just be closed?

I also note by reviewing the pyproject.toml that the extra includes a `tox<4` directive, suggesting the tool may not be compatible with newer versions of tox.

**Comment by DiddiLeija** [comment]
> I see now [tox-to-nox docs](https://nox.thea.codes/en/stable/usage.html#converting-from-tox) direct the user to install the prerequisite extra. So maybe this issue should just be closed?

:+1:

> I also note by reviewing the pyproject.toml that the extra includes a `tox<4` directive, suggesting the tool may not be compatible with newer versions of tox.

IIRC that issue was being addressed in #673 :)
